### PR TITLE
Cuts-from-proofs: Use uint32_t for matrix indexes

### PR DESCRIPTION
**Pull request by aehyvari**
This is what the cuts-from-proofs (after the minor comments) would look like if we used `uint32_t`.

### CODE REVIEW

**Review by blishko — COMMENTED** [review]
I am fine with this change, but the compiler requires more changes.

File: src/tsolvers/lasolver/CutCreator.cc [hunk 1]
```diff
@@ -210,7 +210,7 @@ Representation initFromConstraints(std::vector<CutCreator::DefiningConstaint> co
         }
     }
 
-    auto rows = constraints.size();
+    auto rows = static_cast<uint32_t>(constraints.size());
```
Comment (blishko) [inline src/tsolvers/lasolver/CutCreator.cc:213]:
```suggestion
    uint32_t rows = constraints.size();
```

**Review by blishko — COMMENTED** [review]
Looks good! I would drop the static_cast if compiler allows it.

File: src/tsolvers/lasolver/CutCreator.cc (hunk 1 above)
Comment (aehyvari) [reply src/tsolvers/lasolver/CutCreator.cc]: Sounds good to me.  Hopefully also to all the compilers :)

**Review by blishko — APPROVED** [review]

**Pull request merged**